lib/commands/context.js

```diff
@@ -125,7 +125,7 @@ helpers.startChromedriverProxy = async function (context) {
     // one stopped unexpectedly
     cd.on(Chromedriver.EVENT_CHANGED, (msg) => {
       if (msg.state === Chromedriver.STATE_STOPPED) {
-        this.onChromedriverStop(context);
+        await this.onChromedriverStop(context);
```

Comment:
this await is not in an async function

Reply:
you are right. eslint does not like it anyway, since it is invoked in a callback. shall I define the callback function as asyncchonous as well or remove await?